doc/release-notes/1.5.0.txt

```diff
@@ -84,6 +84,13 @@ Vital Bindings
  * Moved python bindings associated with config from sprokit.pipeline to
    vital.config.
 
+Vital Config
+
+ * Extended the application_config_file_paths() function to list config
```

Comment:
BTW, I keep reading this and wondering if all the (non-code) instances of "config" should be "configuration"...

Similarly, the use of "config" vs. "configuration" in the later documentation doesn't seem entirely consistent.

Reply:
Looks much better; thanks!